src/osdc/Objecter.h

```diff
@@ -1755,8 +1755,8 @@ class Objecter : public md_config_obs_t, public Dispatcher {
 		 map_dne_bound(0) {}
 
     // no copy!
```

Comment:
might want to delete the comment of `// no copy!`, as the code is self-documenting.